src/internal/work/work.go

```diff
@@ -117,11 +118,20 @@ type Master struct {
 	*taskEtcd
 	taskID    string
 	taskEntry *taskEntry
+	ctx       context.Context
```

Comment:
Not a fan of storing contexts.  But you say below this will be replaced by the TaskService, so not a big deal.

Reply:
Yeah, this is going away with the task service.